recipe/meta.yaml

```diff
@@ -10,18 +11,24 @@ source:
   sha256: a84f4bb7c5da1fc396ac3ebf493fa470a2cf208924b4bd9c6321d65d4906df9e
 
 build:
+  number: 1
   noarch: python
-  number: 0
   script: {{ PYTHON }} -m pip install . --no-deps -vv
 
 requirements:
   host:
-    - python >=3.5
+    - python {{ python_min }}
     - pip
+    - setuptools
   run:
-    - python >=3.5
+    - python >={{ python_min }}
 
 test:
+  requires:
+    - pip
+    - python {{ python_min }}
+  commands:
+    - pip check
```

Comment:
Adds `pip check` as mentioned in issue ( https://github.com/conda-forge/pynvml-feedstock/issues/16 ). Also tests with the Python minimum version as suggested by the linter above ( https://github.com/conda-forge/pynvml-feedstock/pull/17#issuecomment-2513231234 )